volumes/miovision/sql/create-table-intersection_movements.sql

```diff
@@ -0,0 +1,7 @@
+SELECT DISTINCT intersection_uid, classification_uid, leg, movement_uid
+	INTO miovision.intersection_movements
```

Comment:
This table does not exist in the API schema but is referenced in zero padding the API TMCs

Reply:
@rickl4 do you think the find invalid movements function should just reference this instead?

Reply:
Yes

Reply:
Fixed by https://github.com/CityofToronto/bdit_data-sources/commit/a76b3abbdb3745dca5a590a69944b771fea0f2fa